examples/migrations/langgraph-checkpoints/artifacts/runs/20260722T210746Z-ab8e8c6f/memanto-roundtrip-okf/sessions/langgraph-migration-20260722t210746z-ab8e8c6f_2026-07-22_unknown_summary.md

```
@@ -0,0 +1,173 @@
+# Session Summary for langgraph-migration-20260722t210746z-ab8e8c6f
+**Session ID:** `unknown`
+
+---
+
+### [2026-07-22 21:07:47] [ARTIFACT] LangGraph transcript: product-launch
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)